packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -68,17 +68,18 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
   let account2Address: string;
   let expectedGasPrice: string;
 
-  const CHAIN_ID = ConfigService.get('CHAIN_ID') || '0x12a';
+  const CHAIN_ID = (ConfigService.get('CHAIN_ID') as string) || '0x12a';
   const requestId = 'rpc_batch1Test';
   const requestIdPrefix = Utils.formatRequestIdMessage(requestId);
-  const requestDetails = new RequestDetails({ requestId: 'rpc_batch1Test', ipAddress: '0.0.0.0' });
+  const requestDetails = JSON.stringify(new RequestDetails({ requestId: 'rpc_batch1Test', ipAddress: '0.0.0.0' }));
```

Comment:
Same question, if there a bug being fixed here?

Reply:
This just removed another warning in the IDE.